qcelemental/models/molecule.py

```diff
@@ -214,8 +214,8 @@ def json_dict(self, *args, **kwargs):
 
 ### Non-Pydantic API functions
 
-    def visualize(self, *, style: Union[str, Dict[str, Any]] = "ball_and_stick",
-                  canvas: Tuple[int, int] = (400, 400)) -> 'py3Dmol.view':
+    def show(self, style: Union[str, Dict[str, Any]] = "ball_and_stick",
+             canvas: Tuple[int, int] = (400, 400)) -> 'py3Dmol.view':
```

Comment:
style/canvas as positional or keyword args?

Reply:
Always keyword, good point.

Dashboard could be good as well over visualize as that is more of what it is moving too.

For `plot` would we want `show_reaction` and `plot_statistics` or both show?

Reply:
I think choose one or the other btwn show & plot. Predictable API more valuable than nature of the viz. "show" is fine with me -- I just think it could be guessed as "print", whereas "plot" definitely suggests additional backend.

I like "dashboard" a little better than "visualize" b/c more concrete. But good with either.

Reply:
> For plot would we want show_reaction and plot_statistics or both show?

Ok, I read this again and realized you were making a distinction btwn chemistry and chemistry+data. That weakens my desire for consistency, but I think I still favor both as show/plot/display.

Reply:
I agree on `show` for both as well. Something that will be changed in other repositories, but worth thinking about here I think.